libs/oeth/swap/src/components/ApyChart.tsx

```diff
@@ -212,74 +213,67 @@ export const ApyChart = (props: StackProps) => {
   );
 
   return (
-    <Card
-      sxCardTitle={{
-        padding: 0,
-        paddingInline: { xs: 2, md: 3 },
-        paddingY: 1.438,
-      }}
-      sxCardContent={{
-        display: 'flex',
-        flexDirection: 'column',
-        gap: 2,
-        paddingInline: 0,
-      }}
-      title={
-        <Stack
-          direction="row"
-          alignItems="center"
-          justifyContent="space-between"
-        >
-          <Typography>
-            {intl.formatMessage({ defaultMessage: 'APY' })}
-          </Typography>
-          <Stack direction="row">
-            {limitOptions.map((d) => (
-              <Button
-                key={`duration-${d?.value ?? 'All'}`}
-                size="small"
-                onClick={() => {
-                  setLimit(d);
-                }}
-                sx={{
-                  minWidth: 0,
-                  width: 40,
-                  color: 'text.primary',
-                  ...(d.value === limit.value && {
-                    backgroundColor: (theme) =>
-                      theme.palette.background.default,
-                  }),
-                }}
-              >
-                {intl.formatMessage(d.label)}
-              </Button>
-            ))}
+    <Card>
+      <CardHeader
+        title={
+          <Stack
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <Typography>
+              {intl.formatMessage({ defaultMessage: 'APY' })}
+            </Typography>
+            <Stack
+              direction="row"
+              sx={{
+                borderRadius: 1,
+                padding: 0.25,
+                backgroundColor: 'grey.700',
+                gap: 0.5,
+              }}
+            >
+              {limitOptions.map((d) => (
+                <Button
+                  key={`duration-${d?.value ?? 'All'}`}
+                  variant="text"
+                  size="small"
+                  onClick={() => {
+                    setLimit(d);
+                  }}
+                  sx={{
+                    color: 'text.secondary',
+                    borderRadius: 1,
+                    minWidth: 0,
+                    width: 40,
+                    ...(d.value === limit.value && {
+                      background: (theme) =>
+                        theme.palette.background.gradientSelected,
+                    }),
+                  }}
+                >
+                  {intl.formatMessage(d.label)}
+                </Button>
+              ))}
+            </Stack>
```

Comment:
The annoying thing about this versus tailwind is how 1 line of code is spread out into several lines of code.
```jsx
          <Stack
            direction="row"
            alignItems="center"
            justifyContent="space-between"
          >
```
versus
```jsx
<div className="flex items-center justify-between">
```

Or
```jsx
            <Stack
              direction="row"
              sx={{
                borderRadius: 1,
                padding: 0.25,
                backgroundColor: 'grey.700',
                gap: 0.5,
              }}
            >
```
versus
```jsx
<div className="flex rounded p-1 bg-gray-700 gap-2">
```

The difference is huge in this example.

Reply:
yeah tailwind is basically syntactic sugar for css while css-in-js engines are closer to original css. The good part is that you have type checking and autocompletion, I usually only type the first 2-3 letters and leave the rest to IDE. For some very repetitive, I even added some vs code user snippets
```
"intl-message": {
    "prefix": "int",
    "body": "intl.formatMessage({ defaultMessage: '' })",
    "description": "react-intl default message"
  },
  "center": {
    "prefix": "cen",
    "body": "display=\"flex\" justifyContent=\"center\" alignItems=\"center\"",
    "description": "flex center"
  }
```